src/ethereum_test_types/eof/v1/constants.py

```diff
@@ -13,7 +13,10 @@
 
 MAX_RETURN_STACK_HEIGHT = 1024
 
-MAX_OPERAND_STACK_HEIGHT = 1023
+MAX_STACK_INCREASE_LIMIT = 0x3FF
```

Comment:
If we want to go with the existing convention that the limit constants start with `MAX_` this can be named `MAX_MAX_STACK_INCREASE`. WDYT?

Reply:
It seems a bit confusing to me tbh.